{"run_number": "1744880186814522", "step_name": "ingenieria_de_datos", "task_id": "2", "attempt_id": 0, "name": "_exception", "content_type": "gzip+pickle-v4", "type": "metaflow.artifact", "sha": "f3887973c2408e7e5746a78c59187138c95a160f", "ds_type": "local", "location": ":root:/home/clau/proyectos/metaflow/Practica-SIE-METAFLOW/.metaflow", "flow_id": "InitFlow", "user_name": "clau", "tags": [], "system_tags": ["runtime:dev", "python_version:3.12.3", "user:clau", "metaflow_version:2.15.6"], "ts_epoch": 1744880189066}